### PR TITLE
feat: add edit cashier component

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Edit Cashier component created.

## Related issues and discussion
#762 

## Screenshots, if any
Screenshot from 2020-05-23 21-32-28

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/organization/tellers/edit-cashier/edit-cashier.component.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            </mat-checkbox>
+          </div>
+
+        </div>
+
+        <!-- <div fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column" style="margin-top:10px;">
+          <div fxFlex="48%" class="password-never-expires-wrapper">
+            <mat-checkbox labelPosition="before" formControlName="active">
+              Active
+            </mat-checkbox>
```
Comment (karantakalkar) [inline src/app/organization/tellers/edit-cashier/edit-cashier.component.html:76]: why commented?

File: src/app/organization/tellers/edit-cashier/edit-cashier.component.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button type="button" mat-raised-button [routerLink]="['../']">Cancel</button>
+        <button mat-raised-button color="primary" [disabled]="!editCashierForm.valid">Submit</button>
+      </mat-card-actions>
+
+    </form>
+
+  </mat-card>
+
+</div>
```
Comment (karantakalkar) [inline src/app/organization/tellers/edit-cashier/edit-cashier.component.html:98]: eof

File: src/app/organization/tellers/cashier-template.resolver.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
```
Comment (karantakalkar) [inline src/app/organization/tellers/cashier-template.resolver.ts:1]: rename as edit cashier resolver and move to edit cashier folder just like for edit office

**Comment by BLasan** [comment]
@karantakalkar Done. Please review

**Comment by karantakalkar** [comment]
WIll be merged once cashier list and view are in place

**Comment by karantakalkar** [comment]
@BLasan remove MCs for review

**Comment by BLasan** [comment]
> @BLasan remove MCs for review

Done

**Comment by karantakalkar** [comment]
@BLasan  resolve please

**Comment by BLasan** [comment]
> @BLasan resolve please

Done

**Comment by BLasan** [comment]
> Thanks

welcome :)

**Pull request merged**